EIPS/eip-1898.md

```diff
@@ -1,39 +1,42 @@
 ---
 eip: 1898
-title: Add `blockHash` to JSON-RPC methods which accept a default block parameter.
+title: Add `blockHash` to defaultBlock methods
+description: Add `blockHash` option to JSON-RPC methods that currently support defaultBlock parameter.
 author: Charles Cooper (@charles-cooper)
+discussions-to: https://ethereum-magicians.org/t/eip-1898-add-blockhash-option-to-json-rpc-methods-that-currently-support-defaultblock-parameter/11757
+status: Review
 type: Standards Track
 category: Interface
-status: Stagnant
 created: 2019-04-01
 requires: 234, 1474
```

Comment:
EIP-1474 needs to be pushed forward, too.

Reply:
opened a separate PR for this https://github.com/ethereum/EIPs/pull/5986